bigtable/use-cases/fraudDetection/src/main/java/com/example/util/UtilFunctions.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.util;
+
+import java.util.ArrayList;
+
+public final class UtilFunctions {
+
+  private UtilFunctions() { }
+
+  /**
+   * @param input ArrayList of Strings
+   * @return comma-seperated string from the input
+   */
+  public static String arrayListToCommasString(final ArrayList<String> input) {
+    StringBuilder mlFeaturesStr = new StringBuilder();
```

Comment:
shouldn't this be called something like stringBuilder? mlFeaturesStr is too specific here